cmd/block-generator/runner/run.go

```diff
@@ -335,7 +336,8 @@ func startIndexer(logfile string, loglevel string, indexerBinary string, algodNe
 		"--postgres", postgresConnectionString,
 		"--server", indexerNet,
 		"--logfile", logfile,
-		"--loglevel", loglevel)
+		"--loglevel", loglevel,
+		"--cpuprofile", cpuprofile)
```

Comment:
Is it disabled when this is an empty string?

Reply:
Yes. I think what happens is, an empty string is treated as one of the command line arguments.